portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +8,41 @@
 #food-container {
   margin-top: 20px;
 }
+
+#navbar {
+  position: fixed;
+  list-style-type: none;
+  top: 0;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color:#2d2075;    
+}
+
+.heading {
+    float:right;
```

Comment:
Extra whitespace in front of float? Does the auto-formatter fix this?

Reply:
auto-formatter has stopped working for me again but I manually fixed it for now